benchmarks/benchmarks/infinite_sheds.py

```diff
@@ -0,0 +1,100 @@
+"""
+ASV benchmarks for infinite_sheds.py
+"""
+
+import numpy as np
+import pandas as pd
+from pvlib.bifacial import infinite_sheds
+from pvlib import location, tracking
+
+
+class InfiniteSheds:
+
+    def setup(self):
+        self.times = pd.date_range(start='20180601', freq='1min',
+                                   periods=8760*25)
```

Comment:
Do we need such a long input for these benchmarks?  The runtimes as reported by the GitHub CI are pretty hefty: 

```
[  1.35%] ··· ...heds.InfiniteSheds.time_get_irradiance_fixed            9.85±0s
[  2.03%] ··· ....InfiniteSheds.time_get_irradiance_poa_fixed            4.90±0s
[  2.70%] ··· ...finiteSheds.time_get_irradiance_poa_tracking            38.8±0s
[  3.38%] ··· ...s.InfiniteSheds.time_get_irradiance_tracking             failed
[  3.38%] ···· asv: benchmark timed out (timeout 60.0s)
```

Reply:
ah yes, didn't mean to leave it so long!